app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -65,18 +79,46 @@ class ExplorationActivityPresenter @Inject constructor(
     this.internalProfileId = internalProfileId
     this.topicId = topicId
     this.storyId = storyId
+    this.explorationId = explorationId
     this.context = context
     this.backflowScreen = backflowScreen
+    if (getExplorationManagerFragment() == null) {
+      val explorationManagerFragment = ExplorationManagerFragment()
+      val args = Bundle()
+      args.putInt(
+        ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY,
```

Comment:
Instead of this you should use key specific to this fragment.  Like:
`ExplorationManagerFragment.EXPLORATION_FRAGMENT_MANAGER_INTERNAL_PROFILE_ID_ARGUMENT_KEY`

Similar change in `ExplorationManagerFragment`

Reply:
Keeping this open so that we can finalise this after team meeting.

My suggestion is this:
in `ExplorationActivityPresenter`
```
   if (getExplorationManagerFragment() == null) {
      activity.supportFragmentManager.beginTransaction().add(
        R.id.exploration_fragment_placeholder,
        ExplorationManagerFragment.newInstance(internalProfileId),
        TAG_EXPLORATION_MANAGER_FRAGMENT
      ).commitNow()
    }
```

In `ExplorationManagerFragment`
```
  companion object {
    internal const val EXPLORATION_MANAGER_FRAGMENT_INTERNAL_PROFILE_ID_ARGUMENT_KEY = "ExplorationManagerFragment.internal_profile_id"

    fun newInstance(internalProfileId: Int): ExplorationManagerFragment {
      val explorationManagerFragment = ExplorationManagerFragment()
      val args = Bundle()
      args.putInt(EXPLORATION_MANAGER_FRAGMENT_INTERNAL_PROFILE_ID_ARGUMENT_KEY, internalProfileId)
      explorationManagerFragment.arguments = args
      return explorationManagerFragment
    }
  }
```

Reply:
Will discuss this in team meeting.

Reply:
This will be included for entire exploration arguments in separate PR.

Reply:
@nikitamarysolomanpvt Please create a issue number accordingly and link it here.

Reply:
Issue created https://github.com/oppia/oppia-android/issues/1267.

Reply:
Per the team meeting, is this something that can be resolved now?

Reply:
We will be creating different PR for entire exploration arguments.